iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/ReceiveMessagesCommon.java

```diff
@@ -235,59 +209,60 @@ public IotHubMessageResult onCloudToDeviceMessageReceived(com.microsoft.azure.sd
         }
     }
 
-    public static class MessageCallbackMqtt implements com.microsoft.azure.sdk.iot.device.MessageCallback
+    protected static boolean hasExpectedProperties(com.microsoft.azure.sdk.iot.device.Message msg, Map<String, String> messageProperties)
```

Comment:
Are we not able to import `com.microsoft.azure.sdk.iot.device.Message` to make this more readable?

Reply:
I can remove a couple cases of this, but this test class needs to reference the ```Message``` class from the device client side and the ```Message``` class from the service client side. Java doesn't support importing classes and renaming them (import com.microsoft.X as Y) to handle cases like this.